bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1068,20 +1098,18 @@ else if (isPlayingStream(currentURI) || isPlayingRadioStartedByAmazonEcho(curren
                         if (response != null) {
                             List<String> fields = SonosXMLParser.getRadioTimeFromXML(response);
 
-                            if (fields != null && !fields.isEmpty()) {
+                            if (!fields.isEmpty()) {
                                 opmlUrlSucceeded = true;
 
                                 resultString = new String();
```

Comment:
```suggestion
                                resultString = "";
```
I do find it strange that this field is only initialized here when later code uses it in a possibly uninitialized state.

Reply:
Changed to "".
`resultString` is initialized to null, this is expected, the idea is to triger a change only when this is really expected.